Update tutorial for new `Copy` button (#135007)
python/cpython: Doc/tutorial/introduction.rst

.. only:: html

   You can use the ``Copy`` button, which strips prompts and omits output, to

>    easily copy and paste the input lines into your interpreter.

Reviewer: Note, that ``Copy`` button is invisible, unless you move mouse pointer over the example box.  I think you should mention that somehow.

Author: I guess so, I added a section.

.. only:: html

>    You can use the ``Copy`` button (it appears in the upper-right corner

Reviewer 2: ```suggestion
   You can use the "Copy" button (it appears in the upper-right corner
```

>    example, and which strips prompts and omits output, to

Reviewer: ```suggestion
   You can use the ``Copy`` button (it appears in the upper-right corner
   when hovering over a code example), which strips prompts and omits output, to
```
Maybe something like this?

Author: I don't have a strong preference, so I'll go with it for now.

>    when hovering over a code example), which strips prompts and omits output, to

Reviewer 2: How do you hover with a touchscreen?

```suggestion
   when hovering or tapping a code example), which strips prompts and omits output, to
```
?

>    easily copy and paste the input lines into your interpreter.

Reviewer 2: It's easy until it's not easy. 
https://developers.google.com/style/word-list#easy 

For example, the copy button works for mouse and touchscreen, it doesn't work for keyboard-only, and possibly not for screen reader. 
```suggestion
   copy and paste the input lines into your interpreter.
```

Author: It is from the existing comment, I see your point.
